aspnetcore/mvc/controllers/filters.md

```diff
@@ -231,23 +188,33 @@ Filters can be added by type or by instance. If an instance is added, that insta
 * An instance is created for each request.
 * Any constructor dependencies are populated by [dependency injection](xref:fundamentals/dependency-injection) (DI).
 
+<!-- Review
+
+Should the following be removed?  Does it add value?
```

Comment:
I think this could be removed. The page has already covered how to add a filter by type.